src/lib/forms/PipelineForm.svelte

```diff
@@ -29,25 +48,11 @@
       props.is_coverage_polygon === true) &&
     props.intervention_type === "area";
 
-  // Sets the intervention name to "From {road1 and road2} to {road3 and
```

Comment:
So for this, I think we settled on leaving the Autofill button, but not filling out the name by default. So I think the change is near https://github.com/acteng/atip/blob/62cd8ea959440f65ebc102612d6ab8b44d434387/src/lib/draw/route/RouteMode.svelte#L27, if the schema is pipeline, don't copy over the route_name we get. Same in EditGeometryMode, it copies route_name sometimes

Reply:
Hmm it's annoying that the RouteMode needs to know about the schema now
I wonder if we can get around it

Reply:
We should probably stick schema in a global store; plumbing it is annoying

Reply:
Still need to revert the changes here -- keep the autofill button, we just want to not populate it by default